docs/sources/flow/reference/components/prometheus.exporter.snmp.md

```diff
@@ -117,6 +117,12 @@ For example, `targets` can either be passed to a `prometheus.relabel`
 component to rewrite the metrics' label set, or to a `prometheus.scrape`
 component that collects the exposed metrics.
 
+The exported targets will use the configured [in-memory traffic][] address
+specified by the [run command][].
```

Comment:
My thought with documenting it was to make sure that people knew it was a special target and not something they could scrape from outside of the agent.

Reply:
It threw me a bit off as the fact that the target is even a network-based thing was an implementation detail that the user didn't need to know, but I get it from that perspective.

Up to you whether you want it or not, it's okay by me.

Reply:
Right now my gut is saying we should leave this in, since it's bordering between being an implementation detail and being something users need to know about, especially if `agent.internal` happens to be a real address on the user's DNS servers.